src/sentry/static/sentry/app/components/organizations/globalSelectionHeader/globalSelectionHeader.jsx

```diff
@@ -181,10 +181,15 @@ class GlobalSelectionHeader extends React.Component {
 
   didQueryChange = (prevProps, nextProps) => {
     const urlParamKeys = Object.values(URL_PARAM);
-    return !isEqual(
-      pick(prevProps.location.query, urlParamKeys),
-      pick(nextProps.location.query, urlParamKeys)
-    );
+    const prevQuery = pick(prevProps.location.query, urlParamKeys);
+    const nextQuery = pick(nextProps.location.query, urlParamKeys);
+
+    // If no next query is specified keep the previous global selection values
+    if (Object.keys(nextQuery).length === 0) {
```

Comment:
If the selection parameters have been removed from the query string isn't that a change?

Reply:
Right now I can't imagine a situation where we link to a global selection page but don't pass any explicit params AND we would want it to reset a store